sycl/include/CL/sycl/detail/pi.h

```diff
@@ -36,10 +36,11 @@
 // 2. A number of types needed to define pi_device_binary_property_set added.
 // 3. Added new ownership argument to piextContextCreateWithNativeHandle.
 // 4. Add interoperability interfaces for kernel.
+// 5. Added new ownership argument to piextQueueCreateWithNativeHandle.
```

Comment:
The numbering (5.) is not in sync with PI versions

Reply:
should it match with the major/minor version? I've followed the inst from line#25 above and increased both by 1.

Reply:
Following instructions in line #25 is all fine. Just let's have the comments about the changes tell what was the version where it was made. So, in your case I'd say have "5" changed to "4.6". And, please add a comment about this numbering in the comment above.

Reply:
done.